pkgs/by-name/le/legcord/package.nix

```diff
@@ -3,18 +3,18 @@
 , fetchFromGitHub
 , pnpm
 , nodejs
-, electron_31
+, electron_32
 , makeWrapper
 , copyDesktopItems
 , makeDesktopItem
 }:
 stdenv.mkDerivation rec {
-  pname = "armcord";
-  version = "3.3.1";
+  pname = "legcord";
+  version = "1.0.0";
 
   src = fetchFromGitHub {
-    owner = "ArmCord";
-    repo = "ArmCord";
+    owner = "Legcord";
+    repo = "Legcord";
     rev = "v${version}";
     hash = "sha256-rCcjanmr4s9Nc5QB3Rb5ptKF/Ge8PSZt0WvgIul3RGs=";
```

Comment:
Just noticed the hashes did not change even though the version and source changed (yet it builds fine). Double checking now.

Reply:
Got the updated hashes, confirmed by building again and validating the armcord -> legcord label changes in the application.

Reply:
Ah yeah good catch - when I built it using `nixpkgs-review`, it just pulled from my local cache. I gotta be more careful with that 👍🏿

Reply:
Sadly it’s not even a locale cache thing; cache.nixos.org will happily give you the old FOD. Sorry for failing to catch this.